src/commands/config.ts

```diff
@@ -187,42 +196,53 @@ export default class ConfigCommand implements ICommand {
 			}
 		}
 
-		if (args.length > 3 && args[0] == 'add') {
+		if (args.length >= 3 && args[0] == 'add') {
 			const key = args[1];
 			const value = args.slice(2).join(" ");
+			if (value == 'null') { 
+				message.reply('No value specified');
+				return;
+			}
 
 			if (key == 'whiteListedGames') {
-				if (value == 'null') {
-					message.reply('No game specified');
+				const twitch = TwitchClient.getInstance()
+				const game = await twitch.getGameData(value);
+				if (!game) {
+					message.reply('Game does not exist for Twitch');
 					return;
-				} else {
-					const twitch = TwitchClient.getInstance()
-					const game = await twitch.getGameData(value);
-					if (!game) { 
-						message.reply('Game does not exist for Twitch');
-						return;
-					}
-					
-					WhiteListedGamesRepository.Add(guildId, game.id, game.name);
 				}
+				WhiteListRepository.AddGame(guildId, game.id, game.name);
+			} else if (key == 'whiteListedRoles') {
+				const role = guild.roles.cache.find(r => r.id === value)
+				if (role === undefined) { 
+					message.reply('Role id does not exist in this guild');
+					return;
+				}
+				WhiteListRepository.AddRole(guildId, role.id, role.name);
 			}
 		}
 
-		if (args.length > 3 && args[0] == 'remove') {
+		if (args.length >= 3 && args[0] == 'remove') {
 			const key = args[1];
 			const value = args.slice(2).join(" ");
+			if (value == 'null') { 
+				message.reply('No value specified');
+				return;
+			}
 
 			if (key == 'whiteListedGames') {
-				if (value == 'null') {
-					message.reply('No game specified');
+				if (wl.games.find(g => g.name == value)) {
+					WhiteListRepository.RemoveGame(guildId, value);
+				} else { 
+					message.reply('Game is not in whitelist');
+					return;
+				}
+			} else if (key == 'whiteListedRoles') {
+				if (wl.roles.find(r => r.id == value)) {
```

Comment:
Other role settings allow for adding by name, would be nice to allow this with whitelisted roles as well

Reply:
Applied it to this as well